src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -24,7 +29,53 @@ else if(e.getActionCommand().equals("Figure Joe 2")) {
 			FiguresJoe.figureScript2(driverManager.getCurrentDriver());
 		}
 		else if(e.getActionCommand().equals("Figure Jane")) {
-			FiguresJoe.figureScript2(new Job2dDrawerAbstractAdapter(driverManager.getCurrentDriver()));
+			FiguresJane.figureScript(new Job2dDrawerAbstractAdapter(driverManager.getCurrentDriver()));
+		}
+		else if(e.getActionCommand().equals("Figure Sebastian 1")) {
+			DriverCommand driverCommandSet1 = new SetPositionCommand(0, 0);
+			DriverCommand driverCommandOperate1 = new OperateToCommand(0, 100);
+			DriverCommand driverCommandOperate2 = new OperateToCommand(100, 50);
+			DriverCommand driverCommandOperate3 = new OperateToCommand(0, 0);
+
+			driverCommandSet1.execute(driverManager.getCurrentDriver());
+			driverCommandOperate1.execute(driverManager.getCurrentDriver());
+			driverCommandOperate2.execute(driverManager.getCurrentDriver());
+			driverCommandOperate3.execute(driverManager.getCurrentDriver());
+		}
+		else if(e.getActionCommand().equals("Figure Sebastian 2")) {
+			ComplexCommand complexCommand = new ComplexCommand();
+			complexCommand.addCommand(new SetPositionCommand(0, 0));
+			complexCommand.addCommand(new OperateToCommand(0, 200));
+			complexCommand.addCommand(new OperateToCommand(100, 200));
+			complexCommand.addCommand(new OperateToCommand(100, 0));
+			complexCommand.addCommand(new OperateToCommand(0, 0));
+
+			complexCommand.execute(driverManager.getCurrentDriver());
+		}
+		else if(e.getActionCommand().equals("Figure Sebastian 3")) {
+			ComplexCommand complexCommand = new ComplexCommand();
+			complexCommand.addCommand(new SetPositionCommand(-100, 0));
+			complexCommand.addCommand(new OperateToCommand(0, -100));
+			complexCommand.addCommand(new OperateToCommand(100, 0));
+			complexCommand.addCommand(new OperateToCommand(0, 100));
+			complexCommand.addCommand(new OperateToCommand(-100, 0));
+
+			complexCommand.execute(driverManager.getCurrentDriver());
```

Comment:
This is not a factory. A factory should have methods that return a driver command (not execute it). Missing point 3.1.5.